Corvus.Extensions.Samples/TraversalExtensionsSample.dib

```diff
@@ -0,0 +1,251 @@
+#!markdown
+
+## Traversal Extensions - Sample Notebook
+
+#!markdown
+
+First, add a reference to the `Corvus.Extensions` NuGet package.
+
+#!csharp
+
+#r "nuget: Corvus.Extensions, 1.1.4"
+
+using Corvus.Extensions;
+
+#!markdown
+
+### ForEachAsync()
+
+#!markdown
+
+Execute an async action for each item in the enumerable.
+
+#!csharp
+
+public async Task<bool> isEqualToOne(int x)
```

Comment:
For me, the fact that your method has a return value is a bit confusing, as the `ForEachAsync` method doesn't require this and won't use the return value at all.

I'd suggest just having your `IsEqualToOne` method return a `Task` rather than `Task<bool>`.